src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -43,6 +44,16 @@ String user = (String) request.getSession().getAttribute("user");
       var chatDiv = document.getElementById('chat');
       chatDiv.scrollTop = chatDiv.scrollHeight;
     };
+
+    // show contents of dropDown menu
+    function showDrop() {
+      var dropDiv = document.getElementById("SettingDropDown");
+      if(dropDiv.style.display === "none") {
```

Comment:
nit: tertiary operators make sense here:
`dropDiv.style.display = dropDiv.style.display === "none" ? "block" : "none";`

It's a matter of personal style though. :)